roles/sap_ha_cluster/vars/RedHat_8.yml

```diff
@@ -0,0 +1,6 @@
+---
+# Overwrite HA_CLUSTER repository ID to use E4S repository
+# - an alternative logic could be to enable the repo before running ha_cluster
```

Comment:
as already done in role sap_general_preconfigure, see https://github.com/sap-linuxlab/community.sap_install/blob/de9379075487d80f2d3a5b68d39165ef27c46802/roles/sap_general_preconfigure/tasks/RedHat/installation.yml#L16 .

Reply:
This role is made independent of the SAP preconfigure/install roles. Also, since the ha_cluster linux system role includes the repository enabling steps, this definition in sap_ha_cluster only ensures that the repository variable that ha_cluster uses is set to the correct repo name (containing e4s). There is no repo enabling done in the sap_ha_cluster role as such, it only translates the variable into ha_cluster to provide the correct name and ensures compatibility with the expected landscape.